Sources/CSG.swift

```diff
@@ -326,10 +326,10 @@ public extension Mesh {
             }
             let rect = Polygon(
                 unchecked: [
```

Comment:
This is the only suspicious place I could find in the change set. @nicklockwood can you try what happens when `-Direction(normal)` is changed to `Direction(-normal)`?